src/main/webapp/mapScript.js

```diff
@@ -89,53 +96,66 @@ function addNewTableItem(name, index) {
   tabel.appendChild(newPlace);
 }
 
-function deletePlace(contentId) {
-  let tableItem = document.getElementById("place" + contentId);
+function getActualIndex(placeIndex) {
+  var actualIndex = 0;
+  for (var i = 0; i < markersArray.length; i++) {
+    if (markersArray[i].data.index == placeIndex) {
+      actualIndex = i;
```

Comment:
nit: it might be more readable to just return i from here, rather than setting it and breaking.

Reply:
Done